shinhye/app.js

```diff
@@ -47,7 +48,135 @@ app.post("/users/signup", async (req, res) => {
     [username, firstName, lastName, age, email, password]
   );
 
-  res.status(201).json({ message: "Successfully created!" });
+  return res.status(201).json({ message: "Successfully created!" });
+});
+
+// add a post
+app.post("/posts", async (req, res) => {
+  const { userID, userProfileImage, postImageURL, postContent } = req.body;
+
+  await AppDataSource.query(
+    `INSERT INTO posts(
+      user_id, 
+      user_profile_image,
+      post_image_url,
+      post_content
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [userID, userProfileImage, postImageURL, postContent]
+  );
+
+  return res.status(201).json({ message: "Successfully created!" });
+});
+
+// view all posts
+app.get("/posts", async (req, res) => {
+  const postData = await AppDataSource.query(
+    `
+    SELECT
+    user_id as userID,
+    user_profile_image as userProfileImage,
+    id as postId,
+    post_image_url as postImageUrl,
+    post_content as postContent
+    FROM posts;
+    `
+  );
```

Comment:
쿼리문 포매팅을 수정해주세요!
적절한 들여쓰기와 구조적 통일성을 통해 가독성을 향상시키고 코드를 유지보수하기도 용이해집니다.
```suggestion
  const postData = await AppDataSource.query(`
    SELECT
      user_id as userID,
      user_profile_image as userProfileImage,
      id as postId,
      post_image_url as postImageUrl,
      post_content as postContent
    FROM posts;
    `);
```

Reply:
넵 반영했습니다!